topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -39,86 +45,244 @@ Sequence quality control is therefore an essential first step in your analysis.
->    > * Copy the link location
->    > * Open the Galaxy Upload Manager
->    > * Select **Paste/Fetch Data**
->    > * Paste the link into the text field
->    > * Press **Start**    
->    {: .tip}
->
->    > ### {% icon tip %} Tip: Change the file type `fastq` to `fastqsanger` once the data file is in your history
->    >
->    > * Click on the pencil button displayed in your data file in the history
->    > * Choose **Datatype** on the top
->    > * Select `fastqsanger`
->    > * Press **save**
->    {: .tip}
+>    {% include snippets/create_new_history.md %}
+>    {% include snippets/rename_history.md %}
+>
+> 2. Import `GSM461178_untreat_paired_subset_1.fastq` from [Zenodo](https://zenodo.org/record/61771) or from the data library (ask your instructor)
+>
+>    ```
+>    https://zenodo.org/record/61771/files/GSM461178_untreat_paired_subset_1.fastq
+>    ```
+>
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+>
+>    As default, Galaxy takes the link as name, so rename them.
+>
+> 4. Rename the file to `reads_1`
+>
+>    {% include snippets/rename_dataset.md %}
+>
+{: .hands_on}
+
+We just imported in Galaxy a file. This file is similar to a file we could get directly from a sequencing facility: a FastQ file.
+
+> ### {% icon hands_on %} Hands-on: Inspect the FastQ file
+>
+> 1. Inspect the file by clicking on the {% icon galaxy-eye %} (eye) icon
 >
->    > ### {% icon comment %} Comments
->    >
->    > Rename the dataset to "First dataset"
->    {: .comment}
-> By default, when data is imported via its link, Galaxy names it with its URL.
 {: .hands_on}
 
-# Quality check
+Although it looks complicated (and maybe it is), the FastQ format is easy to understand with a little decoding. 
+
+Each read, representing a fragment of the library, is encoding by 4 lines:
+
+Line  | Description
+--- | ---
+1 | Always begins with `@` and then information about the read
+2 | The actual nucleic sequence
+3 | Always begins with a `+` and sometimes the same info in line 1
+4 | Has a string of characters which represent the quality scores associated to each base of the nucleic sequence; must have same number of characters as line 2
+
+So for example, the first sequence in our file is:
+
+```
+@SRR031716.1 HWI-EAS299_4_30M2BAAXX:3:1:944:1798 length=37
+GTGGATATGGATATCCAAATTATATTTGCATAATTTG
++SRR031716.1 HWI-EAS299_4_30M2BAAXX:3:1:944:1798 length=37
+IIIIIIIIIIIIIIIIIIIIIIIIIIIII8IIIIIII
+```
+
+It means that the fragment named `SRR031716.1` correspond to the DNA sequence `GTGGATATGGATATCCAAATTATATTTGCATAATTTG` and this sequence has been sequenced with a quality `IIIIIIIIIIIIIIIIIIIIIIIIIIIII8IIIIIII`.
+
+But what does this quality mean?
+
+The quality for each sequence is a string of characters, one for each base of the nucleic sequence, used to characterize the probability of mis-indentification of each base. The score is encoding using the ASCII character table (with some historical diffences):
```

Comment:
What about adding a link to the Wikipedia FASTQ page (https://en.wikipedia.org/wiki/FASTQ_format) or somewhere that has some more info on the historical differences for anyone interested?
It might also be worth saying explicitly that the current lllumina (1.8+) uses Sanger format (Phred+33) but if working with older datasets users may encounter the older scoring schemes. Also may be worth mentioning that FastQC can be used to try to determine what type of quality encoding is used (through assessing the range of Phred values seen in the FASTQ). Could be worth adding a question on that or pointing out that it's available in the FastQC report.